### PR TITLE
lesson-1-1

### PR DESCRIPTION
**Pull request by yasi44**
pull request for lesson-1-1

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+  {
+    id:3,
+    title:"Call mom"
+  }
+]; 
+
 function App() {
+  const listItems = todoList.map(function(item){
+    return <li>{item.id} {item.title}</li>;
+  });
```
Comment (sinancengiz) [inline src/App.js]: item.id should be used as key, and item text should only contain title. 

doc in the link has some examples for how to setting key in li element https://legacy.reactjs.org/docs/lists-and-keys.html

**Review by sinancengiz — CHANGES_REQUESTED** [review]
Great Job! only thing is that key needs to be set in li items , Please request another review when changes are done

**Review by yasi44 — COMMENTED** [review]
key added to <li> tag. pls review.

**Review by sinancengiz — APPROVED** [review]
LGTM! Great Job

**Pull request merged**